### PR TITLE
Hands Gizmos in Editor (Re-added)

### PR DESCRIPTION
**Pull request by njbrown**
Add support for hands in editor
hands are serialized/deserialized to/from scene file

### CODE REVIEW

File: src/editor/handgizmo.cpp [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+		device->setShaderUniform("u_color", color);
+		device->setShaderUniform("u_fresnelPow", fresnelPow);
+	}
+};
+
+HandGizmoHandler::HandGizmoHandler()
+{
+	handScale.setToIdentity();
+	handScale.scale(0.01f);// scale by 1/100;
```
Comment (iKlsR) [inline src/editor/handgizmo.cpp]: Is this to accomodate the c4d exporter? Can't we use collada instead or maybe try experimental gltf? Very bizarre.

File: src/editor/handgizmo.cpp [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+void HandGizmoHandler::loadAssets(const iris::ContentManagerPtr& content)
+{
+	handModel = content->loadModel(IrisUtils::getAbsoluteAssetPath("app/models/right_hand_anims.fbx"));
+	handMaterial = QSharedPointer<iris::Material>(new HandGizmoMaterial());
+}
+
+bool HandGizmoHandler::doHandPicking(const iris::ScenePtr& scene,
+	const QVector3D& segStart,
+	const QVector3D& segEnd,
+	QList<PickingResult>& hitList)
```
Comment (iKlsR) [inline src/editor/handgizmo.cpp]: Like lists huh?

File: src/io/scenewriter.cpp [hunk 3]
```diff
@@ -383,6 +387,23 @@ void SceneWriter::writeParticleData(QJsonObject& sceneNodeObject, iris::Particle
...
+	sceneNodeObject["guid"] = node->getGUID();
+	switch (node->handPose->getPoseType()) {
+	case iris::HandPoseType::Grab:
+		sceneNodeObject["poseType"] = "grab";
+		break;
+	case iris::HandPoseType::Pinch:
+		sceneNodeObject["poseType"] = "pinch";
+		break;
+	default:
+		sceneNodeObject["poseType"] = "grab";
```
Comment (iKlsR) [inline src/io/scenewriter.cpp]: nitpick, can move grab to last and remove the break to fall though to default.

File: src/editor/handgizmo.cpp (hunk 1 above)
Comment (njbrown) [reply src/editor/handgizmo.cpp]: The scale of the model coming the oculus website is too large

**Pull request merged**